### ISSUE TITLE
Display problem

### ISSUE THREAD

**Issue by ConSolatory** [Wacky-Mole/WackyEpicMMOSystem#16]
Hello . The latest version has a problem with displaying skills.
When increasing skills, a window appears 
On the left side + Adding different components
On the right side is the total count of certain parameters (which are shown in visual mode, which may be added). === it is precisely this display function that does not change (previously it was known in advance how the choice of one or another parameter would affect it - now it is static).
But ! If you apply the points from the left panel, they will be correct. 
Bottom line. Fix the right parameter panel (displaying parameters before they are applied).

Thank you.
